### ISSUE TITLE
Cache values for tszipped files

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/tsbrowse#113]
I think there may be a problem with caching data frames for tszipped files, as there might not be ``file_uuid`` on the ts returned by ``tszip.load``. We should check if this is the case. 

If not, it's a tszip issue.

**Comment by benjeffery** [comment]
No longer relevant after #167
